### PR TITLE
Fixes + instructions for translations

### PR DESCRIPTION
**Pull request by oscfdezdz**
I copied Notejot's `po/README.md` file although I only added one section, hope it's ok 😄 

I'll add Spanish translation once #4 is fixed.

### CODE REVIEW

**Comment by lainsce** [comment]
Something got broken....

File: data/ui/window.ui [hunk 1]
```diff
@@ -159,7 +159,7 @@
                             <property name="margin-top">30</property>
                             <property name="margin-start">30</property>
                             <property name="margin-bottom">20</property>
-                            <property name="label" translatable="true">Palettes</property>
+                            <property name="label" translatable="yes">Palettes</property>
```
Comment (lainsce) [inline data/ui/window.ui:162]: This should be "true" not "yes". "yes" isn't a thing. :P

Comment (oscfdezdz) [reply data/ui/window.ui:162]: Other lines are `translatable="yes"` tagged 🙃

**Comment by oscfdezdz** [comment]
> Something got broken....

From CI output, it seems default repo branch it's not defined.

File: data/ui/window.ui (hunk 1 above)
Comment (lainsce) [reply data/ui/window.ui:162]: Oh.

**Comment by lainsce** [comment]
Try doing a new commit after pulling the fix for that :)

**Comment by oscfdezdz** [comment]
> Try doing a new commit after pulling the fix for that :)

Done 😄

**Comment by lainsce** [comment]
I see the CI error is outside this PR, so I'm merging it now

**Pull request merged**